WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -769,6 +782,7 @@ private extension ZendeskUtils {
         static let deviceFreeSpace: UInt64 = 360000089123
         static let networkInformation: UInt64 = 360000086966
         static let logs: UInt64 = 22871957
+        static let currentSite: UInt64 = 360000103103
```

Comment:
the way Zendesk requires us to specify this is so bizarre!

Reply:
Actually, this is down to iOS. If these are defined as `NSNumber` it fails on 32-bit devices. This would be a good point for an explanatory comment though. I'll do that before I merge.